xml/System.Threading.Tasks/Parallel.xml

```diff
@@ -950,6 +950,7 @@
   
  Any methods in the source orderable partitioner return <see langword="null" />.</exception>
         <related type="Article" href="https://docs.microsoft.com/previous-versions/msp-n-p/ff963552(v=pandp.10)">Parallel Loops</related>
+        <exception cref="T:System.AggregateException">The exception that is thrown to contain an exception thrown from one of the specified delegates.</exception>
```

Comment:
```suggestion
```

Reply:
Did you intend to remove this?

Reply:
Yes, because I was moving this line to before the related tag,